src/mesh/RadioInterface.cpp

```diff
@@ -462,6 +462,14 @@ void RadioInterface::applyModemConfig()
         cr = loraConfig.coding_rate;
         bw = loraConfig.bandwidth;
 
+        if (cr == 0) // This parameter is not an integer
+            cr = 5; // An invalid cr/sf will cause the rx1262 init to bail partway through with an error. Which is bad because not
+                    // all registers may be inited.
+        if (sf == 0)
+            sf = 11;
+        if (bw == 0)
+            bw = 250; // Just guess and use long fast settings
```

Comment:
Maybe also carry over the ternary logic for checking if we're on 2.4G lora?
![image](https://github.com/meshtastic/firmware/assets/9000580/e480084d-0f22-4e77-be5d-cb2d4d0d88b7)

Reply:
I don't think that's super critical - because as long as these initial values are at least acceptable for the hw, later in the boot we'll be plugging in the 'real' settings based on the channel.